src/api_test.ts

```diff
@@ -71,19 +65,16 @@ Deno.test("SlackAPI class", async (t) => {
               });
             });
 
-            await assertRejects(
-              async () => {
-                return await client.apiCall("chat.postMessage", {});
-              },
-              (error: Error) => {
-                assertInstanceOf(error, HttpError);
-                if (isHttpError(error)) {
-                  assertEquals(error.headers?.get("Retry-After"), "120");
-                  assertEquals(error.status, 429);
-                  assertEquals(error.message, "429: ratelimited");
-                }
-              },
-            );
+            try {
+              await client.apiCall("chat.postMessage", {});
+            } catch (error) {
+              assertInstanceOf(error, HttpError);
+              if (isHttpError(error)) {
```

Comment:
This conditional should always return true. If it doesn't, then the previous line's assertion would fail. It's main purpose in this block is for type narrowing; using `isHttpError` narrows the error type and allows for referencing e.g. `headers` and `status` on the error object.